app/assets/stylesheets/rides.scss

```diff
@@ -1,3 +1,75 @@
 // Place all the styles related to the Rides controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+@mixin round-button {
+  height: 120px;
+  width: 120px;
+  border-radius: 65px;
+  padding: 0px;
+  font-size: 1.3em;
+  border: 1px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.